### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by marcelozarate** [mdn/mdn-community#233] [Content] [answered]
Hi! I've tried searching around but couldn't find any traces of hacktoberfest taking part at content or translated-content apart from [this](https://github.com/mdn/translated-content/issues/2692). Is there any reason why this didn't happen? If there are no issues (pun intended) with it, it would be a perfect time to label issues, generate some more, and spread the word looking for new collaborators and sharing the spirit of OSS.

Hope you don't mind pings @mdn/localization-team-leads @mdn/yari-content-es @mdn/yari-content-fr @mdn/yari-content-pt-br @mdn/yari-content-ru @mdn/yari-content-ja @mdn/yari-content-ko @mdn/yari-content-zh @mdn/yari-content-mdn 

https://hacktoberfest.com/participation/#maintainers

**Comment by marcelozarate** [comment]
First we will need to add the [hacktoberfest topic to the repo](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics)

Then add two new labels _hacktoberfest_ and _hacktoberfest-accepted_ and label some good first issues.

Finally we need to register before October 31st at https://hacktoberfest.com/

The reviews and accepting is as usual, just tagging with _hacktoberfest-accepted_ when the PR is good to merge.

**Reply by SphinxKnight** [reply]
OK for me, I can provide/create a few issues for `l10n-fr`
@mdn/localization-team-leads and @mdn/mdn-community-engagement what's your opinion here?

**Reply by marcelozarate** [reply]
Thanks @SphinxKnight, l10n-es also already committed for creation/tagging of issues.

**Reply by sashasushko** [reply]
Unfortunately, I have no power to do this right now for @mdn/yari-content-ru. But I think it's a great idea. And if someone from RU-team ready to do this it will be great.

**Comment by schalkneethling** [comment, marked as answer]
@marcelozarate Thank you so much for reaching out and for your enthusiasm 🎉  The community is more than welcome to rally around this. The core staff and our partners are not quite ready to handle an influx of new pull requests at the moment. The good news is that we are getting our processes more well defined and freeing up our time using automation.

This initiative is something we are seriously considering for 2023. Happy coding!
